topics/metagenomics/tutorials/foodborne-pathogen-detection-nanopore-sequencing-data/tutorial.md

```diff
@@ -0,0 +1,1669 @@
+In this tutorial, the datasets are all generated by Biolytix. They are all chicken spiked with _Salmonella's_ different stains. The two barcodes we are using are; Barcode 10 Spike 2, which is chicken spiked with _Salmonella enterica subsp. enterica_ this dataset was generated on the 14th of July 2022, and Barcode 11 Spike 2b, which is chicken spiked with _Salmonella enterica subsp. houtenae_ generated on 20th of July 2022.
+
+![Biolytix workflow for sequencing ONT datasets](./images/Biolytix_Workflow_Sequencing.png "The Biolytix workflow for spiking food e.g. chicken, meat, etc. with known pathogens e.g. Salmonella then preform Nanopore sequencing datasets with built-in base caller")
+
+# Create your histories and Get Data
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create two new histories for this tutorial one for every sampling time
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Rename the histories to `Barcode10`and `Barcode11`
+>
+>    {% snippet faqs/galaxy/histories_rename.md %}
+>
+> 3. Import the samples Fastq.qz files via link from [Zenodo]({{ page.zenodo_link }}) or Galaxy shared data libraries to each history respectively:
+>
+>    - `Barcode10Spike2`: to `Barcode10` history
+>
+>    ```text
+>    {{ page.zenodo_link }}/files/Barcode10_Spike2.fastq.gz
+>    ```
+>
+>    - `Barcode11Spike2b`: to `Barcode11` history
+>
+>    ```text
+>    {{ page.zenodo_link }}/files/Barcode11_Spike2b.fastq.gz
+>    ```
+>
+>    - From the shared data libraries in Galaxy, datasets can be also found under __Shared Data__ menu (top panel) then __Data libraries__, __Biolytix__, __Spiked__, __Salmonella__ then either __Spike2__, __Spike 2 Barcode 10 - 15 collapsed__ or __Spike2b__, __Spike2b Barcode 10-15 and unclassified Collapsed__
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+>
+{: .hands_on}
+
+# Choose your own tutorial
+{% include _includes/cyoa-choices.html option1="ShortVersion" option2="LongVersion" default="ShortVersion" %}
+
+<div class="ShortVersion" markdown="1">
+
+As you chose to follow the short version of the tutorial we will be running the five sub-workflows each for a complete task. Starting with pre-processing of the sequenced datasets till the pathogen tracking among all our chosen samples. In order to check the step by step tools used for every sub-workflow please follow the tutorial's LongVersion.
+
+# Pre-Processing
+
+Before starting any analysis, it is always a good idea to assess the quality of your input data and improve it when possible by trimming and filtering reads.
+
+In this section we will run a sub-workflow that performs the following tasks with the following tools:
+1. Assess the reads quality before and after improving it using [__FastQC__](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/), [__NanoPlot__](https://github.com/wdecoster/NanoPlot) and  [__MultiQC__](https://multiqc.info/)
```

Comment:
I added and revised everything, only this comment I will work on tomorrow morning :)

Reply:
But I will need help with that, since sometime I dont find any paper to the tool I find only a link to it or to its documentation which the ones already included

Reply:
If there is no paper in the tool help, then no need to add the reference